skimage/_shared/utils.py

```diff
@@ -237,13 +237,21 @@ def convert_to_float(image, preserve_range):
         using img_as_float. Also see
         https://scikit-image.org/docs/dev/user_guide/data_types.html
 
+    Notes:
+    ------
+    * Input images with `float32` data type are not upcasted.
```

Comment:
```suggestion
    * Input images with `float32` data type are not upcast.
```

See https://www.usingenglish.com/reference/irregular-verbs/cast.html

=)